x-pack/test_serverless/api_integration/test_suites/observability/threshold_rule/avg_pct_fired.ts

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
@fkanout Is there another ticket to add these tests in stateful too? Since all of our users are using stateful, it is more important to have test coverage there compared to serverless.

Reply:
If I recall correctly, yes, we have one. But before that, I would like to check if the work about reusing tests across ENVs is done.

Reply:
Can you please share the ticket here? :)